<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorCodeCompletionWidget</class>
 <widget class="QWidget" name="EditorCodeCompletionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="grpEnabled">
     <property name="title">
      <string>Enable code competion</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Minimum id length to show completion </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinMinCharRequired">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkEditorShareCodeParser">
        <property name="text">
         <string>Editors share one code parser</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkClearWhenEditorHidden">
        <property name="text">
         <string>Clear all parsed symbols when editor is hidden</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkShowSuggestionWhileTyping">
        <property name="text">
         <string>Show completion suggestions while typing</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Engine options</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="chkParseLocalFiles">
           <property name="text">
            <string>Scan local header files</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkParseSystemFiles">
           <property name="text">
            <string>Scan system header files</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkShowKeywords">
        <property name="text">
         <string>Show keywords in suggestions</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkShowCodeIns">
        <property name="text">
         <string>Show code snippets in suggestions</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkAppendFunc">
        <property name="text">
         <string>Append () when complete functions</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkIgnoreCases">
        <property name="text">
         <string>Ignore case when search suggestions</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkSortByScope">
        <property name="text">
         <string>Prefer local symbols</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkHideSymbolsStartWithUnderline">
        <property name="text">
         <string>Hide symbols start with underscore</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkHideSymbolsStartWithTwoUnderline">
        <property name="text">
         <string>Hide symbols start with two underscores</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="chkRecordUsage">
           <property name="text">
            <string>Prefer symbols mostly used</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnClearUsageData">
           <property name="text">
            <string>Clear usage data</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Completion suggestion window width:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinWidth">
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Completion suggestion window height:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinHeight">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>grpEnabled</tabstop>
  <tabstop>spinMinCharRequired</tabstop>
  <tabstop>chkEditorShareCodeParser</tabstop>
  <tabstop>chkClearWhenEditorHidden</tabstop>
  <tabstop>chkShowSuggestionWhileTyping</tabstop>
  <tabstop>chkParseLocalFiles</tabstop>
  <tabstop>chkParseSystemFiles</tabstop>
  <tabstop>chkShowKeywords</tabstop>
  <tabstop>chkShowCodeIns</tabstop>
  <tabstop>chkAppendFunc</tabstop>
  <tabstop>chkIgnoreCases</tabstop>
  <tabstop>chkSortByScope</tabstop>
  <tabstop>chkHideSymbolsStartWithUnderline</tabstop>
  <tabstop>chkHideSymbolsStartWithTwoUnderline</tabstop>
  <tabstop>chkRecordUsage</tabstop>
  <tabstop>btnClearUsageData</tabstop>
  <tabstop>spinWidth</tabstop>
  <tabstop>spinHeight</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
